src/screens/OrganizationDashboard/OrganizationDashboard.tsx

```diff
@@ -229,12 +237,17 @@ function organizationDashboard(): JSX.Element {
                         <h6>No posts present</h6>
                       </div>
                     ) : (
-                      postData?.postsByOrganization.slice(0, 5).map((post) => {
+                      sortAndSlicePostsByCreatedAt(
```

Comment:
Similarly, Why are posts sorted here on the client-side?

You can use `postsByOrganizationConnection` query to fetch posts from backend itself sorted by `createdAt` field.